### PR TITLE
[ISSUE #3978]Remove unnecessary annotations in TopicCreateRequest and add unit test.

### LINKED ISSUE #3978: [Enhancement] Remove unnecessary annotations in TopicCreateRequest and add unit test.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

org.apache.eventmesh.admin.rocketmq.request.TopicCreateRequest

* The field name is the same with `@JsonProperty`'s value.
* `super()` in this constructor will be invoked by default.
* Unit test case is not yet done.

### Describe the solution you'd like

* Remove `@JsonProperty`.
* Remove `super()`.
* Add unit test case.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by pandaapo**
Fixes #3978.

### Motivation

org.apache.eventmesh.admin.rocketmq.request.TopicCreateRequest

* The field name is the same with `@JsonProperty`'s value.
* `super()` in this constructor will be invoked by default.
* Unit test case is not yet done.

### Modifications

* Remove `@JsonProperty`.
* Remove `super()`.
* Add unit test case.

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by xwm1992** [comment]
@pandaapo please fix the ci check error.

**Comment by pandaapo** [comment]
> @pandaapo please fix the ci check error.

OK now. I will check all my other PRs and fix if necessary.

**Review by xwm1992 — DISMISSED** [review]

**Review by Alonexc — DISMISSED** [review]

File: eventmesh-admin/eventmesh-admin-rocketmq/src/main/java/org/apache/eventmesh/admin/rocketmq/request/TopicCreateRequest.java [hunk 1]
```diff
@@ -30,16 +30,13 @@ public class TopicCreateRequest {
 
     @JsonCreator
     public TopicCreateRequest(@JsonProperty("name") String topic) {
-        super();
         this.name = topic;
```
Comment (lrhkobe) [inline eventmesh-admin/eventmesh-admin-rocketmq/src/main/java/org/apache/eventmesh/admin/rocketmq/request/TopicCreateRequest.java:33]: Maybe, using the same variable name for topic and name would improve readability.

Comment (pandaapo) [reply eventmesh-admin/eventmesh-admin-rocketmq/src/main/java/org/apache/eventmesh/admin/rocketmq/request/TopicCreateRequest.java:33]: I'm not sure. Maybe the author plan to add more fields to this class in the future.
Will it be more readable than now when more fields are added to this class?

Comment (xwm1992) [reply eventmesh-admin/eventmesh-admin-rocketmq/src/main/java/org/apache/eventmesh/admin/rocketmq/request/TopicCreateRequest.java:33]: change the `name` to `topic` is better

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**